dev/reference/tools/pd-control.md

```diff
@@ -222,11 +226,11 @@ The configuration above is global. You can also tune the configuration by config
 > **Note:**
 >
 > The configuration of the namespace only supports editing `leader-schedule-limit`, `region-schedule-limit`, `replica-schedule-limit` and `max-replicas`.
-
-    ```bash
-    >> config set namespace ts1 leader-schedule-limit 4 // 4 tasks of leader scheduling at the same time at most for the namespace named ts1
-    >> config set namespace ts2 region-schedule-limit 2 // 2 tasks of region scheduling at the same time at most for the namespace named ts2
-    ```
+>
+> ```bash
+> >> config set namespace ts1 leader-schedule-limit 4 // 4 tasks of leader scheduling at the same time at most for the namespace named ts1
+> >> config set namespace ts2 region-schedule-limit 2 // 2 tasks of region scheduling at the same time at most for the namespace named ts2
+> ```
```

Comment:
Thanks for updating the Note format. However, code blocks within a Note often fail to display well. If you want to check whether this one can work, make sure check it on the PingCAP website after this PR is merged.